app.py

```diff
@@ -119,15 +119,40 @@ def skill():
     Handles Skill requests
     '''
     if request.method == 'GET':
-        return jsonify({})
+        return data['skill'], 200
```

Comment:
Please review issue #12 again. You must use return a skill given an index

Reply:
I actually return a skill given an index in the `a_skill` function

Reply:
My bad! Working from my phone so I overlooked that. Is there a way to combine the two into one function? Do you have any plans for the initial function?

Reply:
Okay, I will look into this ASAP